test/test_cpp_extensions_open_device_registration.py

```diff
@@ -0,0 +1,90 @@
+# Owner(s): ["module: cpp-extensions"]
+
+import os
+import shutil
+import sys
+
+import torch.testing._internal.common_utils as common
+import torch
+import torch.utils.cpp_extension
+from torch.utils.cpp_extension import CUDA_HOME, ROCM_HOME
+
+
+TEST_CUDA = torch.cuda.is_available() and CUDA_HOME is not None
+TEST_CUDNN = False
+TEST_ROCM = torch.cuda.is_available() and torch.version.hip is not None and ROCM_HOME is not None
+if TEST_CUDA and torch.version.cuda is not None:  # the skip CUDNN test for ROCm
+    CUDNN_HEADER_EXISTS = os.path.isfile(os.path.join(CUDA_HOME, "include/cudnn.h"))
+    TEST_CUDNN = (
+        TEST_CUDA and CUDNN_HEADER_EXISTS and torch.backends.cudnn.is_available()
+    )
+IS_WINDOWS = sys.platform == "win32"
+
+
+def remove_build_path():
+    if sys.platform == "win32":
+        print("Not wiping extensions build folder because Windows")
```

Comment:
print?

Reply:
I copied this from an existing cpp extension test we have, but agreed - will clean up